.gitignore

```diff
@@ -1,3 +1,3 @@
 phpunit.xml
 vendor/
-
+.idea/
```

Comment:
this should be ignored using a local rule rather than a project rule, as it is specific to your setup